frontend/src/components/Banner.test.tsx

```diff
@@ -52,8 +61,23 @@ describe('Banner', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('does not show "Refresh" button if mode is "info"', () => {
+    const tree = shallow(
+      <Banner
+        message={'Some message'}
+        mode={'info'}
+        refresh={() => {
+          /* do nothing */
+        }}
+      />,
+    );
+    expect(tree.find('WithStyles(Button).refreshButton').at(0)).toMatchInlineSnapshot(`null`);
```

Comment:
Hmmm, the selector is kind of hacky. What if RefreshButton later changed implementation, so it doesn't show as `WithStyles(Button)`.

Suggest finding the button by its text content: https://github.com/Bobgy/pipelines/blob/20157593093b888538095bdb2560c36b1354038e/frontend/src/components/NewRunParameters.test.tsx#L48-L52

also use `.toBeNull()` instead of match inline snapshot to be more explicit.